ajax/libs/ng-jsoneditor/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ng-jsoneditor",
+  "filename": "ng-jsoneditor.min.js",
+  "version": "1.5.1",
+  "description": "Angular version of the insanely cool jsoneditor",
+  "keywords": [
+    "jsoneditor",
+    "angular",
+    "ng"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/angular-tools/ng-jsoneditor.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/angular-tools/ng-jsoneditor.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "ng-jsoneditor*!*(.min.min*)"
```

Comment:
The current pattern can't get the files from all versions.
Let's use `ng-jsoneditor!(*min.min*)*` to get the files.
Thanks.

PS. Exclude `1.0.0/ng-jsoneditor.min.min.js` because it's the same with `1.0.0/ng-jsoneditor.min.js`
```
md5sum ajax/libs/ng-jsoneditor/1.0.0/*
854d98066dd380aef5c38afdcb4ed179  ajax/libs/ng-jsoneditor/1.0.0/ng-jsoneditor.js
aa1c9efa3a83e6497448846d929da964  ajax/libs/ng-jsoneditor/1.0.0/ng-jsoneditor.min.js
aa1c9efa3a83e6497448846d929da964  ajax/libs/ng-jsoneditor/1.0.0/ng-jsoneditor.min.min.js
```